windows/ReactNativeWebView/ReactWebViewManager.cpp

```diff
@@ -67,9 +66,9 @@ namespace winrt::ReactNativeWebView::implementation {
                         if (srcMap.find("__packager_asset") != srcMap.end()) {
                             isPackagerAsset = srcMap.at("__packager_asset").AsBoolean();
                         }
-
-                        if (isPackagerAsset && uriString.find("assets") == 0) {
-                            uriString.replace(0, 6, "ms-appx://");
+                        if (isPackagerAsset && uriString.find("file://") == 0) {

```

Comment:
During debug, this path could be the dev server, e.g. http://localhost:8081/...
Is it possible to handle that case as well?

Reply:
There are 3 main things that it will come through. 

It comes in as 
Debug:
* NonBundle: 
  * http://localhost:8081/assets/src/components/Screens/NoticesScreen/notices.html?platform=windows&hash=4f3f61dcf9ff4c24e6fa59c905ad1276 
* Bundle: 
  * file://assets/src/components/Screens/NoticesScreen/notices.html
* RelaseBundle
* ms-appx-web://Bundle/assets/src/components/Screens/NoticesScreen/notices.html

Reply:
`if (isPackagerAsset && uriString.find("file://") == 0) {`

in non bundle (i.e. debug)  the isPackagerAsset will be false and it wont go thru this path.